tools/topology/platform/intel/intel-generic-dmic-kwd.m4

```diff
@@ -81,6 +81,7 @@ PIPELINE_PCM_ADD(sof/pipe-`DMICPROC'-capture.m4,
 undefine(`PGA_NAME')
 undefine(`PIPELINE_FILTER1')
 undefine(`PIPELINE_FILTER2')
+undefine(`DMICPROC')
```

Comment:
this one toom should be in the first patch

Reply:
fixed. thanks.